web/src/components/upload/drop.utils.js

```diff
@@ -143,3 +145,11 @@ export const extractFileMetaContent = async (file, endLine = 10) => {
     return [""];
   }
 };
+
+/**
+ * Retrieve all of the devices
+ * @returns {Array<Device>}
+ */
+export const getDevices = async () => {
```

Comment:
As this is a function that we use in `index.js`, we'll want to create an `utils.js` to keep its utility functions. We can then place this function there. (The lack of sub-name for `utils.js` indicates that it belongs to the default file- `index.js`)

As it is a unit function, we can also add unit tests. However, we don't have the proper node modules for that, yet. So these tests can wait for another day.